app/config/sources.inc.php

```diff
@@ -103,7 +103,7 @@
 // Default priority is 3
 $mozillaorg_lang = [
     'download_button.lang' => [
-        'deadline'          => '2016-04-29',
+        'deadline'          => '2016-05-24',
```

Comment:
I would put a much shorter deadline for this, e.g. 2017-05-16. Note that we're in 📅 though ;-)